.changeset/weak-roses-laugh.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit-html': patch
+'lit': patch
+---
+
+Add an additional security brand check to CompiledTemplates; Similar to [#2307](https://github.com/lit/lit/pull/2307)
```

Comment:
note that this is a security fix that may be breaking, but only to an API that most likely no one is using because it's the output of a compiler that we haven't released yet

Reply:
Please see new changeset message. You were fast on the review!

Reply:
Nice!